components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpointTest.java

```diff
@@ -602,6 +604,9 @@ public void testAuthorizeForAuthenticationResponse(boolean isResultInRequest, bo
 
             Response response = oAuth2AuthzEndpoint.authorize(httpServletRequest, httpServletResponse);
             assertEquals(response.getStatus(), expected, "Unexpected HTTP response status");
+            if (!isAuthenticated) {
+                assertTrue(response.getEntity().toString().contains(OAuthConstants.OAuth20Params.STATE));
```

Comment:
shall we check if the same state value (https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1731/files#diff-1172a5d982d509df8c84f7b2595af92e3d874d0e7c5c290b2b3d61003ba28741R249) is there in the response?